src/tests/ov_helpers/ov_snippets_models/src/subgraph_fq.cpp

```diff
@@ -59,20 +59,16 @@ std::shared_ptr<ov::Model> ThreeFQFunction::initReference() const {
                                                                               std::vector<float>{127},
                                                                               ov::element::i8);
     auto fq1 = ov::builder::subgraph::makeFakeQuantizeTypeRelaxed(fq0, ov::element::f32, fq1_data);
-    auto subgraph0 = std::make_shared<ov::snippets::op::Subgraph>(NodeVector{data0},
-                                          std::make_shared<ov::Model>(NodeVector{fq1}, ParameterVector{indata0}));
-
-    auto indata1 = std::make_shared<op::v0::Parameter>(precision, subgraph0->get_shape());
     auto fq2_data = ov::builder::subgraph::FakeQuantizeOnDataWithConstant(256,
                                                                               std::vector<ov::Shape>{{1, 3, 1, 1}, {1, 3, 1, 1}, {1}, {1}},
                                                                               std::vector<float>{-98.16883, -142.82466, -155.0642700},
                                                                               std::vector<float>{97.334106, 141.599884, 153.8145446},
                                                                               std::vector<float>{0},
                                                                               std::vector<float>{255},
                                                                               ov::element::u8);
-    auto fq2 = ov::builder::subgraph::makeFakeQuantizeTypeRelaxed(indata1, ov::element::f32, fq2_data);
-    auto subgraph1 = std::make_shared<ov::snippets::op::Subgraph>(NodeVector{subgraph0},
-                                      std::make_shared<ov::Model>(NodeVector{fq2}, ParameterVector{indata1}));
+    auto fq2 = ov::builder::subgraph::makeFakeQuantizeTypeRelaxed(fq1, ov::element::f32, fq2_data);
```

Comment:
Some ops are not tokenized into one subgraph, simply because the limitation of number of grps https://github.com/openvinotoolkit/openvino/blob/master/src/common/snippets/src/pass/collapse_subgraph.cpp#L614.  This test case is corresponding to such scenario. Since now the number of available registers will be set from the plugin side, the limitation from the snippets library side is removed https://github.com/openvinotoolkit/openvino/pull/23180#discussion_r1535086546. The ref subgraph should be updated accordingly. Thanks Ivan!